plugins/PrivacyManager/vue/src/Compliance/Compliance.vue

```diff
@@ -9,15 +9,74 @@
   <h2>
     <EnrichedHeadline>{{ translate('PrivacyManager_Compliance') }}</EnrichedHeadline>
   </h2>
+
+  <p>
+    {{ translate('PrivacyManager_ComplianceSelectSite') }}
+  </p>
+
+  <SiteSelector
+    id="complianceDashboard"
```

Comment:
Checked other uses of SiteSelector and they all have the id because they are also preceded by a <label> tag (being a form field of sorts), I reckon we should also add a label above the field, amend the id so it's clearer it's a site selector, e.g. complianceSite.